js/ui/map.js

```diff
@@ -985,6 +985,21 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Returns a Boolean indicating whether the map is currently in a "moving" state.
+     * Returns `false` if zooming, rotating, pitching, and dragging are all inactive.
+     *
+     * @returns {boolean} A Boolean indicating whether the map is moving.
+     */
+    isMoving: function() {
+        if (this.zooming ||
+            this.rotating ||
+            this.pitching ||
+            this.dragPan._active ||
+            this.dragRotate._active) return true;
```

Comment:
Good questions! I'm not familiar enough with these interactions enough to know the differences. The only way I was able to capture "pan" and "rotate" actions was through those `dragPan` and `dragRotate` parameters, though. Are there other places to look for them?

> Should Map#isMoving return true if the user starts a "drag rotate" interaction but doesn't actually rotate the map? 

I'm not sure - any chance you could provide a case where this happens?